lib/reddex/links/link.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Reddex.Links.Link do
```

Comment:
Modules should have a @moduledoc tag.

Reply:
@Math3v i'd say it's fine. the comment stays here even if the credo doesn't show it on localhost.